src/dlz_aws.cpp

```diff
@@ -196,6 +213,22 @@ isc_result_t dlz_lookup(
     }
   }
 
+  if (StringUtils::CaselessCompare(state->autoscaler_zone_name.c_str(), zone)) {
+    std::string clientAddr;
+    std::vector<std::string> nodes;
+    get_src_address(methods, clientinfo, &clientAddr);
+    if (state->client->TryResolveAutoscaler(name, clientAddr, &nodes)) {
```

Comment:
Is "name" everything under the zone, or just the first level?

e.g. if I want to set the dns-alias tag to "general.kafka" on the ASG, will "general.kafka.asg.aws.tcdc.io" resolve correctly?